### DISCUSSION TITLE
usage of CURLOPT_LOW_SPEED_LIMIT and CURLOPT_LOW_SPEED_TIME in multi curl.

### DISCUSSION THREAD

**Discussion by Harishreddy01** [curl/curl#8905] [Q&A] [answered]
I am trying to use  CURLOPT_LOW_SPEED_LIMIT and CURLOPT_LOW_SPEED_TIME provided API's and not able to suceessfully hit these limits. I am using multi curl and set these options on every easy handle we attach to multi handle. From code i see "Curl_speedcheck" is called if state is "MSTATE_RATELIMITING" from multi_runsingle function. Do we need to enable any other options to use CURLOPT_LOW_SPEED_LIMIT and CURLOPT_LOW_SPEED_TIME.

**Comment by bagder** [comment, marked as answer]
No, they should just work when set.
